packages/core/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@department-of-veterans-affairs/component-library",
   "description": "VA.gov component library. Includes React and web components.",
-  "version": "29.0.0",
+  "version": "30.0.0",
```

Comment:
Updating the core version because we will create a release after this PR is merged.